.github/workflows/test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - uses: ankane/setup-mongodb@v1
       - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 24
```

Comment:
I prefer using the oldest supported Node.js version in CI. It’s easy to verify compatibility with the latest version, but I never test with the oldest one locally. This approach ensures the version we claim to support actually works.